airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -802,20 +1281,33 @@ definitions:
         type: string
         enum: [ListPartitionRouter]
       cursor_field:
+        title: Cursor Field
+        description: While iterating over list values, the name of field used to reference a list value.
         type: string
+        examples:
+          - "section"
+          - "{{ config['section_key'] }}"
       values:
+        title: Values
+        description: The list of attributes being iterated over and used as input for the requests made to the source API.
         anyOf:
           - type: string
           - type: array
             items:
               type: string
+        examples:
+          - ["section_a", "section_b", "section_c"]
+          - "{{ config['sections'] }}"
       request_option:
+        title: Request Option
```

Comment:
This is called `Inject partition value into outgoing HTTP request` in the UI

Reply:
updated